packages/engine-server/src/DendronEngineV3.ts

```diff
@@ -341,8 +343,46 @@ export class DendronEngineV3 implements DEngine {
     throw Error("querySchema not implemented");
   }
 
-  async queryNotes(): ReturnType<DEngineClient["queryNotes"]> {
-    throw Error("queryNotes not implemented");
+  /**
+   * See {@link DEngine.queryNotes}
+   */
+  async queryNotes(
+    opts: QueryNotesOpts
+  ): ReturnType<DEngineClient["queryNotes"]> {
+    const ctx = "Engine:queryNotes";
+    const { qs, vault, onlyDirectChildren, originalQS } = opts;
+
+    // Need to ignore this because the engine stringifies this property, so the types are incorrect.
```

Comment:
Hm - where is this stringify happening?

Reply:
Also copied over lol. I'll leave this for now and do some cleanup as follow through